app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -188,6 +189,63 @@ ControllerContainer::InitializeBeam() {
   }
 }
 
+void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex, std::vector<vrb::Matrix>& jointTransforms)
+{
+    if (!m.Contains(aControllerIndex))
+        return;
+
+    Controller &controller = m.list[aControllerIndex];
+    if (!controller.beamParent)
+        return;
+
+    // Initialize hand joints if needed
+    if (controller.handJointTransforms.size() == 0) {
+        controller.handJointTransforms.resize(jointTransforms.size());
+
+        CreationContextPtr create = m.context.lock();
+        controller.handMeshToggle = Toggle::Create(create);
+        controller.beamParent->AddNode(controller.handMeshToggle);
+
+        ProgramPtr program = create->GetProgramFactory()->CreateProgram(create, 0);
+        RenderStatePtr state = RenderState::Create(create);
+        state->SetProgram(program);
+        vrb::Color handColor = vrb::Color(0.0f, 0.50f, 0.0f);
```

Comment:
I do really think we should use a gray here, any other color is really distracting, and I think it fits what Meta is using to draw hands. I'll use 0.3f for the 3 components.